language-design.md

```diff
@@ -51,6 +137,13 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
+"Every API is a little language, and people must learn to read and write it" [Bloch, 2006]. Thus, 
+we must be sure that APIs (along with other languages), are as easy to understand as possible, 
+especially as "it should rarely require documentation to read code written to a good API" [Bloch, 
+2006]. If someone were to use something like gray(100) without checking the documentation, they 
+might not realize that it actually gives them white [Verou, 2014]; when it comes to an API, this 
+could be a dangerous, rather than an aesthetic, error.
+
```

Comment:
I like this point! I took a different approach to this question, so I hadn't considered the danger of bugs in an API vs the color of something, but you're right that it has a different scale of consequences.